buildconfig.js

```diff
@@ -10,10 +10,13 @@ export default {
   // 'BuildConfig.templateSelection.faq': 1,
   // 'BuildConfig.templateSelection.sponsor': 1,
   // 'BuildConfig.templateSelection.footer': 1,
-  // 'FeatureFlags.faqFlag': false,
-  // 'FeatureFlags.mentorFlag': false,
-  // 'FeatureFlags.registerationFlag': false,
-  // 'FeatureFlags.rsvpOpenFlag': false,
-  // 'FeatureFlags.sponsorFlag': false,
-  // 'FeatureFlags.volunteerFlag': false,
+  // 'featureFlags.faqFlag': false,
+  // 'featureFlags.mentorFlag': false,
+  // 'featureFlags.registerationFlag': false,
+  // 'featureFlags.rsvpOpenFlag': false,
+  // 'featureFlags.sponsorFlag': false,
+  // 'featureFlags.volunteerFlag': false,
+  // 'WebsiteData.Intro.top': 'string',
+  // 'WebsiteData.Intro.left': 'string',
+  // 'WebsiteData.Intro.right': 'string',
```

Comment:
These will crash cms without creating some of the other fields. Check https://dev.nwplus.io and select cmd-f

See if your internal dev partner can help you fix it.

Reply:
I've changed it to `StaticData` to avoid crashing, but let me know if there's a better option for it!